docs/source/gbd2017_models/causes/cause_model_template.rst

```diff
@@ -276,6 +276,325 @@ Cause Model Diagram
+
+**SIR Model**
+
+.. list-table:: State Definitions
+   :widths: 5 5 20
+   :header-rows: 1
+
+   * - State
+     - State Name
+     - Definition
+   * - S
+     - Susceptible
+     - Susceptible to {cause name}
+   * - I
+     - Infected
+     - Infected with {cause name}
+   * - R
+     - Recovered
+     - Infected with {cause name}
+
+
+**With-Condition and Free of Condition Model**
+
+.. list-table:: State Definitions
+   :widths: 1, 5, 10
+   :header-rows: 1
+
+   * - State
+     - State Name
+     - Definition
+   * - C
+     - With **C**\ ondition
+     - Born with {cause name}
+   * - F
+     - **F**\ ree of Condition
+     - Born without {cause name}
+
+Include states, their names and definitions appropriate to your model.
+
+States Data
+"""""""""""
+
+This table contains the **measures** and their **values** for each state in cause-model diagram. This information is used to 
+initialize the model. The common measures in each state are prevalence, birth prevalence, excess mortality rate and disability weights. 
+Cause specific mortality rate is the common measure for all states. In most of the models either prevalence or birth prevalence is used. 
+But in some rare cases like neonatal models both prevalence and birth prevalence are used in model initialization. The Value column contains the formula to calculate 
+the measure in each state.
+
+The structure of the table is as below. For each state, the measures and values must be included.
+
+.. list-table:: States Data
+   :widths: 20 25 30 30
+   :header-rows: 1
+   
+   * - State
+     - Measure
+     - Value
+     - Notes
+   * - State
+     - prevalence
+     - 
+     - 
+   * - State
+     - birth prevalence
+     - 
+     - 
+   * - State
+     - excess mortality rate
+     - 
+     - 
+   * - State
+     - disabilty weights
+     - 
+     -
+   * - ALL
+     - cause specific mortality rate
+     - 
+     - 
+
+An example of SI model with both prevalence and birth prevalence in the initialization is given below to explain better. 
+
+
+.. list-table:: States Data
+   :widths: 20 25 30 30
+   :header-rows: 1
+   
+   * - State
+     - Measure
+     - Value
+     - Notes
+   * - S
+     - prevalence
+     - 1-prevalence_cid
+     - 
+   * - S
+     - birth prevalence
+     - 1-birth_prevalence_cid
+     - 
+   * - S
+     - excess mortality rate
+     - 0
+     - 
+   * - S
+     - disabilty weights
+     - 0
+     -
+   * - I
+     - prevalence
+     - prevalence_cid
+     - 
+   * - I
+     - birth prevalence
+     - birth_prevalence_cid
+     - 
+   * - I
+     - excess mortality rate
+     - :math:`\frac{\text{deaths_cid}}{\text{population} \times \text{prevalence_cid}}`
+     - = (cause-specific mortality rate) / prevalence
+   * - I
+     - disability weights
+     - :math:`\displaystyle{\sum_{s\in \text{sequelae_cid}}} \scriptstyle{\text{disability_weight}_s \,\times\, \text{prevalence}_s}`
+     - = average disability weight over all sequelae
```

Comment:
if you mean average disability weight over all sequelae, then you need to divide by sum(prevalence_s). Otherwise the formula means total disability weight over all sequelae in this disease state.

Reply:
if you add this denominator to your equation, the more precise definition is prevalence weighted average disability weight over all sequelae

Reply:
I think the formula is correct. So I will change the wording as per your suggestion